src/Http/Exception/RedirectException.php

```diff
@@ -31,15 +29,8 @@
  * Additional headers can also be provided in the constructor, or
  * using the addHeaders() method.
```

Comment:
Not sure if we should update or just remove this and let people use the HttpException api naturally.

Reply:
We could deprecated `addHeaders()`. I am not quite sure how common it is to add new headers after the exception instance has been constructed.